src/api/polygons.spec.ts

```diff
@@ -8,33 +7,43 @@ describe('TargomoClient polygon service', () => {
     const sources = [{ lng: 13.3786431, lat: 52.4668237, id: 1}]
 
     const result = <any>await testClient.polygons.fetch(sources, {
+      serializer: 'geojson',
       travelType: 'walk',
       travelEdgeWeights: [300, 600],
-      serializer: 'geojson',
       buffer: 0.00001,
     })
 
     expect(result).toBeDefined()
     expect(result.type).toEqual('FeatureCollection')
 
     const result2 = <any>await testClient.polygons.fetch(sources, {
+      serializer: 'geojson',
       travelType: 'walk',
       travelEdgeWeights: [300, 600],
-      serializer: 'geojson',
       buffer: 0.00001,
     })
 
     expect(JSON.stringify(result)).toEqual(JSON.stringify(result2))
 
     const result3 = <any>await testClient.polygons.fetch(sources, {
```

Comment:
dont use `<any>` you now have typed results, so you should check the return types in the tests